src/VisualStudio/VisualBasic/Impl/BasicVSResources.Designer.vb

```diff
@@ -91,6 +91,24 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
             End Get
         End Property
         
+        '''<summary>
+        '''  Looks up a localized string similar to Arithmetic operators:   ^   *   /   \   Mod   +   -   &amp;.
```

Comment:
Calling `&` arithmetic is... interesting. I know what you mean though... It's probably fine.